klog.go

```diff
@@ -1265,6 +1276,47 @@ func Infof(format string, args ...interface{}) {
 	logging.printf(infoLog, logging.logr, format, args...)
 }
 
+// InfoS structured logs to the INFO log.
+// The msg argument used to add constant description to the log line.
+// The key/value pairs would be join by "=" ; a newline is always appended.
+//
+// Basic examples:
+// >> klog.InfoS("Pod status updated", "pod", "kubedns", "status", "ready")
+// output:
+// >> I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kubedns" status="ready"
+func InfoS(msg string, keysAndValues ...interface{}) {
+	b := &bytes.Buffer{}
+	if infos(b, logging.logr, msg, keysAndValues) {
+		logging.println(infoLog, logging.logr, b)
+	}
+}
+
+// infos if logr is specified, no need to output with logging module
+func infos(b *bytes.Buffer, logr logr.InfoLogger, msg string, keysAndValues []interface{}) bool {
```

Comment:
Some notes about making your code better blend with rest of klog.

Could you move this function to `loggingT` struct?
This would make code more unified as all other methods (`Infof`, `Errorf`) just call `logging` struct.

I would also rename it to `printS` so it matches other functions on `loggingT`

Reply:
done